rule_test.go

```diff
@@ -1,34 +0,0 @@
-package ldclient
```

Comment:
All these assertions got moved to the test_data.json file so we can reuse them in other sdks.